docker-ee-for-windows/install.md

```diff
@@ -0,0 +1,115 @@
+---
+description: How to install Docker EE for Windows
+keywords: windows, windows server, install, download
+title: Install Docker Enterprise Edition for Windows Server 2016
+---
+
+Docker Enterprise Edition for Windows Server 2016 (*Docker EE*) enables native
+Docker containers on Windows Server 2016. The Docker EE installation package includes everything you need to run Docker on Windows Server 2016 systems. This topic describes pre-install considerations, and how to download and install Docker EE.
```

Comment:
nit: this is a long line. It's helpful for code review later if we can keep close to 80 columns per line.